### PR TITLE
Edit Admin rolebinding to add jenkins serviceaccount

### PR DESCRIPTION
**Pull request by cmoulliard**
- [x] Have you created an [issue](https://github.com/redhat-kontinuity/catapult/issues) and used it in the commit message?
https://issues.jboss.org/browse/OBST-219
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/redhat-kontinuity/catapult/pulls) for the same issue?
- [x] Have you built the project locally prior to submission with `mvn clean install`?
Tested locally using minishift and this project "https://github.com/cmoulliard/spring-boot-configmap-booster"

### CODE REVIEW

**Comment by cmoulliard** [comment]
> Will that work in OSO free too?

If the user has the appropriate rights (= basic_user), that should be the case. To be tested

File: services/openshift-service-impl/src/main/java/io/openshift/appdev/missioncontrol/service/openshift/impl/fabric8/openshift/client/Fabric8OpenShiftServiceImpl.java [hunk 1]
```diff
@@ -310,6 +310,17 @@ private void configureProject(final OpenShiftProject project, final InputStream
                             ((OpenShiftProjectImpl) project).addResource(resource);
                         });
             }
+
+            // Add Admin role to the jenkins serviceaccount
+            log.info("Adding role admin to jenkins serviceaccount for project '" + project.getName() + "'");
```
Comment (gastaldi) [inline services/openshift-service-impl/src/main/java/io/openshift/appdev/missioncontrol/service/openshift/impl/fabric8/openshift/client/Fabric8OpenShiftServiceImpl.java]: This should be log.finest

Comment (cmoulliard) [inline services/openshift-service-impl/src/main/java/io/openshift/appdev/missioncontrol/service/openshift/impl/fabric8/openshift/client/Fabric8OpenShiftServiceImpl.java]: log.info is used to log all the info within this method. So, do you want to change it also for the others log. ?

Comment (gastaldi) [inline services/openshift-service-impl/src/main/java/io/openshift/appdev/missioncontrol/service/openshift/impl/fabric8/openshift/client/Fabric8OpenShiftServiceImpl.java]: Yes

**Comment by cmoulliard** [comment]
This error is reported during launchpad-mission creation of the project -> https://gist.github.com/cmoulliard/ca3961800d66cfa5e58e510c01094c47#file-gistfile1-txt-L23

**Comment by cmoulliard** [comment]
Ticket updated : https://issues.jboss.org/browse/OBST-219?focusedCommentId=13396171&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13396171

**Comment by cmoulliard** [comment]
@gastaldi, @ALRubinger launchpad-mission control running locally using WildFlySwarm has been able:
- To create a git project of the booster SpringBoot ConfigMap,
- Create an OpenShift project (= zz1)
- Create from the jenkins-ephemral template the jenkins (master/slave) pod
- Launch the pipeline as defined within the project
- To finally build & run the SpringBoot configMap on https://api.free-int.openshift.com

https://www.dropbox.com/s/kuoedd1s05rqdaj/Screenshot%202017-04-21%2008.51.57.png?dl=0

http http://spring-boot-configmap-zz1.d800.free-int.openshiftapps.com/api/greeting
HTTP/1.1 200
Cache-control: private

{
    "content": "Hello, World from Kubernetes ConfigMap!",
    "id": 7
}

Remark : No permissions issues have been reported

**Comment by cmoulliard** [comment]
Feedback about the PR ?

**Comment by gastaldi** [comment]
Merged based on your feedback

**Comment by cmoulliard** [comment]
Do you know when this change will be part of the next release ?

**Comment by gastaldi** [comment]
When we release the next version, but you can test with master:
https://raw.githubusercontent.com/openshiftio/launchpad-templates/master/openshift/launchpad-template.yaml


Em 23 de abr de 2017 07:58, "Charles Moulliard" <notifications@github.com>
escreveu:

**Pull request merged**